src/vue/common/CollectionLoader.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div class="collection-loader">
+    <button
+      v-if="!isCollectionFetched"
+      class="collection-loader__more-button"
+      @click="loadNextPage">
+      {{ 'common.button-text-more' | globalize }}
+    </button>
+  </div>
+</template>
+
+<script>
+/**
+ * CollectionLoader is responsible for downloading pageable collections.
+ * After loading emits the event ('first-page-load'
+ * - when loaded first page and 'next-page-load' - when load next page)
+ * with the loaded data and decide whether or not it should still show itself.
+ *
+ * @prop {Function} firstPageLoader - function for downloading first page of
+ * data
+ * @prop {Function} pageLimit - quantity request per page
```

Comment:
the maximum quantity of records per page

Reply:
ok